### PR TITLE
feature: add jvm support

### PR DESCRIPTION
**Pull request by osugikoji**
## :loudspeaker: Type of change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
Add JVM support

## :green_heart: How did you test it?
Tested on the JVM sample created

## :crystal_ball: Next steps
Generate a new release

### CODE REVIEW

File: samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/vertex")
+public class VertexController {
+
+    @Autowired
+    private VertexService vertexService;
+
+    @GetMapping("prediction")
```
Comment (xyzwilliamxyz) [inline samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java:23]: This mapping requires `/` as a prefix.

File: samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/vertex")
+public class VertexController {
+
+    @Autowired
```
Comment (xyzwilliamxyz) [inline samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java:18]: it's usually recommended to inject dependencies through constructor as it reduces the dependency from framework when doing unit tests.

**Review by xyzwilliamxyz — APPROVED** [review]

File: samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java (hunk 1 above)
Comment (osugikoji) [reply samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java:23]: The @RequestMapping annotation already defines the base URL mapping for the class, so it doesn't need to explicitly specify the "/" as a prefix in the `getMapping`

File: samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java (hunk 2 above)
Comment (osugikoji) [reply samples/jvm/src/main/java/com/hexascribe/vertexai/controller/VertexController.java:18]: as it's just a sample I wasn't paying attention to it, but I totally agree! Made the adjustments

**Pull request merged**